src/Frontend/Page/Static.hs

```diff
@@ -48,8 +52,9 @@ instance Page PageStaticTermsOfUse where
 
 instance ToHtml PageStaticTermsOfUse where
     toHtmlRaw = toHtml
-    toHtml p = semanticDiv p $ do
-        h1_ "Nutzungsbedingungen"
-        p_  "Lorem ipsum"
-        p_  "Duis aumet"
-        p_  "Ut wisi"
+    toHtml p@(PageStaticTermsOfUse terms) = semanticDiv p .
+        div_ [class_ "text-markdown"] $
+            terms ^. html
+
+termsOfUse :: ActionM m => m PageStaticTermsOfUse
+termsOfUse = PageStaticTermsOfUse <$> query Persistent.termsOfUse
```

Comment:
more general remark: this module makes a good case against structuring the page modules along the sections "template" and "page".  we should have sections for pages, and those sections should contain first the page instances and then the handler.